distributionviewer/core/static/js/app/components/views/example-chart.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { LineChart } from 'rd3';
+
+export default function(props) {
+  var exampleData = [{
```

Comment:
This is a great demo of how it will work, we'll just need to figure out how the real data will look and chart it via metrics-graphics next.

I'll work on the api/reducer/store portion then.